File: app/src/main/java/com/semanienterprise/softcom/models/PagesItem.java
package com.semanienterprise.softcom.models;

import android.os.Parcel;
import android.os.Parcelable;

import java.io.Serializable;
import java.util.List;
import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PagesItem implements Serializable {

	@SerializedName("label")
	private String label;

	@SerializedName("sections")
	private List<SectionsItem> sections;

    @Override
 	public String toString(){
		return 
			"PagesItem{" + 
			"label = '" + label + '\'' + 
			",sections = '" + sections + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/semanienterprise/softcom/models/PagesItem.kt
package com.semanienterprise.softcom.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class PagesItem : Serializable {

    @SerializedName("label")
    var label: String? = null
        set(label) {
            field = this.label
        }

    @SerializedName("sections")
    var sections: List<SectionsItem>? = null
        set(sections) {
            field = this.sections
        }

    override fun toString(): String {
        return "PagesItem{" +
                "label = '" + this.label + '\''.toString() +
                ",sections = '" + this.sections + '\''.toString() +
                "}"
    }
}